core/src/main/scala-2.13+/cats/data/NonEmptyLazyList.scala

```diff
@@ -445,6 +445,21 @@ class NonEmptyLazyListOps[A](private val value: NonEmptyLazyList[A])
   final def toNes[B >: A](implicit order: Order[B]): NonEmptySet[B] =
     NonEmptySet.of(head, tail: _*)
 
+  /**
+   * Creates new `NonEmptyVector`, similarly to List#toVector from scala standard library.
+   *{{{
+   * scala> import cats.data._
+   * scala> import cats.instances.int._
+   * scala> val nel = NonEmptyLazyList.fromLazyListPrepend(1, LazyList(2,3,4))
+   * scala> val expectedResult = NonEmptyVector.fromVectorUnsafe(Vector(1,2,3,4))
+   * scala> val result = nel.toNev
+   * scala> result === expectedResult
+   * res0: Boolean = true
+   *}}}
+   */
+  final def toNev[B >: A]: NonEmptyVector[B] =
+    NonEmptyVector.fromVectorUnsafe(toLazyList.toList.toVector)
```

Comment:
does `LazyList` not have its own `toVector` method?

Reply:
You are right. It does. Additionally, I was missing an import. I will push changes. Thank you @LukaJCB :)

Reply:
Of course :)